ISSUE_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Pull Request
+about: Submit a Pull Request & help us improve :-)
+title: "[PULL_REQUEST]"
+labels: Type:Pull_Request
+assignees: ''
+
+---
+
+## How to open a great Pull Request
+
+### What kind of PR is this?
+
+- Please make sure you've read and understood our [contributing guidelines](https://substrafoundation.github.io/src/contribute/CONTRIBUTING.html)
+- Please label this pull request according to the type of issue you are addressing using these [recommended labels](https://substrafoundation.github.io/src/contribute/CONTRIBUTING.html#type-of-issue-and-issue-state): `feature`, `bugfix`, `linting`, `refactoring`, `documentation`, `other`.
```

Comment:
Since we're moving to doc.substra.ai maybe we should update those URLs (and others if there are still references to substrafoundation.github.io)